truss/cli/cli.py

```diff
@@ -64,6 +64,23 @@
     ]
 }
 
+HELLO_PY = """import random
```

Comment:
considered putting this is the "examples" folder...I'm concerned that if we move the examples/change the file structure, we'll break the "init" command. What we run the risk of is that this init diverges from the spelled out example. I think that is ok, but I can also add tests that assert that loading the `hello.py` file works. LMK WYT

Reply:
talked offline - it is okay that these are different